trade-finance-manager-ui/server/helpers/string.test.js

```diff
@@ -42,33 +41,4 @@ describe('helpers - string', () => {
       expect(result).toEqual(false);
     });
   });
-
-  describe('isAlphanumeric', () => {
```

Comment:
Please revert, so the function can be used in future if and when required.

Reply:
(See conversation in `../helpers/string.test.js`)